Fabric.Authorization.API/Services/IdentitySearchService.cs

```diff
@@ -39,21 +39,29 @@ public class IdentitySearchService
             }
 
             var client = await _clientService.GetClient(request.ClientId);
-            var clientRoles = await _roleService.GetRoles(client);
+            var clientRoles = _roleService.GetRoles(client).Result.ToList();
```

Comment:
was there an issue with using await here?

Reply:
@kpaul No, I just needed to ToList it to prevent the "possible multiple enumerations" warning.

Reply:
@bsmithvt - can you ToList() clientRoles later? Removing the await keyword and adding the .Result will cause this function to run synchronously I believe, and may introduce deadlock issues.